packages/uiweb/src/lib/components/space/helpers/space.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Get Status of the Space.
+ * @param {string | Date} datestring - The JavaScript Date String or Date Object.
+ * @returns {string} - The Status of the Space.
+ */
+
+export function getSpaceStatus(datestring: Date | string) {
+    const date = new Date(datestring);
+    const now = new Date();
+    const diff = date.getTime() - now.getTime();
+    const diffDays = diff / (1000 * 3600 * 24);
+  
+    if (diffDays < 0) {
+      return 'Ended';
+    } else if (diffDays > 0) {
+      return 'Scheduled';
+    } else {
+      return 'Live';
+    }
+  }
```

Comment:
eof newline, these errors will lead to linting issues later so please correct them